projectfiles/Xcode/wesnoth_codesign.sh

```diff
@@ -0,0 +1,23 @@
+VERSION=1.13.12
+IN_FILENAME="Wesnoth_${VERSION} rw.dmg"
+OUT_FILENAME="Wesnoth_${VERSION}.dmg"
+VOLUME="/Volumes/Wesnoth ${VERSION}/"
+CODESIGN_IDENTITY='Wesnoth, Inc'
+
+set -euo pipefail
```

Comment:
-o pipefail is a bash feature. Either make it a bash script or just remove. There isn't any pipeline in this script anyway.

Reply:
done.